Userland/js.cpp

```diff
@@ -159,19 +159,96 @@ static void print(JS::Value value)
     putchar('\n');
 }
 
+String sanitize_repl_inputs(String input)
+{
+    StringBuilder result;
+    for (char c : input) {
+        if (c != ' ' && c != '\n') {
+            result.append(c);
+        }
+    }
+    return result.to_string();
+}
```

Comment:
This function seems very limiting; what if I want to `.load` a file that has a space in the name?

Reply:
That's a good point. I am not sure how I would implement that as it stands since I split at spaces in order to differentiate file names. I could require that file names be quoted but that feels a little clunky. What do you think?

Reply:
Actually maybe comma separating them might be nice.